resources/assets/coffee/react/beatmap-discussions-history/main.coffee

```diff
@@ -42,6 +42,7 @@ export class Main extends React.PureComponent
 
     discussions = @state.discussions
     users = @state.users
+    relatedDiscussions = @state.relatedDiscussions
```

Comment:
you can reduce state modification by cloning them, I guess (with `[..]`).

Reply:
Should I change for the the other state too?

Reply:
yes, might as well